### PR TITLE
clarify the confusing value

### PR DESCRIPTION
**Pull request by MaratMussabekov**
#5758

### CODE REVIEW

File: windows/security/identity-protection/hello-for-business/hello-hybrid-aadj-sso-base.md [hunk 1]
```diff
@@ -290,6 +290,8 @@ A **Trusted Certificate** device configuration profile is how you deploy trusted
 5. In the **Enterprise Root Certificate** blade, click **Assignments**.  In the **Include** tab, select **All Devices** from the **Assign to** list.  Click **Save**.
 ![Intune Profile assignment](images/aadj/intune-device-config-enterprise-root-assignment.png)
 6. Sign out of the Microsoft Azure Portal.
+> [!NOTE]
+> After the creation, the **supported platform** parameter of the profile will have value "Windows 8.1 and later", as the certificate configuration for Windows 8.1 and Windows 10 is the same.
```
Comment (illfated) [inline windows/security/identity-protection/hello-for-business/hello-hybrid-aadj-sso-base.md]: Looks quite good, just needs a tiny bit of grammar adjustment.
```suggestion
> After the creation, the **supported platform** parameter of the profile will contain the value "Windows 8.1 and later", as the certificate configuration for Windows 8.1 and Windows 10 is the same.
```

**Comment by mypil** [comment]
@JohanFreelancer9 - kindly do the copy editor review for this PR. Thanks.

**Comment by mypil** [comment]
@mapalko @Dansimp - PR has been copy edited and is ready for final review. This is based on issue 5758.

Thank you.

cc: @kenwith

**Comment by mypil** [comment]
@Dansimp @kenwith - PR has been approved by @mapalko. Can you please assist to merge this PR?

Thank you.

**Comment by mypil** [comment]
@denisebmsft @Dansimp @kenwith - Would any of you be able to merge this PR so we can also close the issue raised? This PR has already been approved by @mapalko.

Thank you.

**Pull request merged**